### PR TITLE
fix: fetch up to 50 live broadcast from YouTube

### LINKED ISSUE #30: Only first 5 broadcasts are listed
According to https://developers.google.com/youtube/v3/live/docs/liveBroadcasts/list#parameters:
> The maxResults parameter specifies the maximum number of items that should be returned in the result set. Acceptable values are 0 to 50, inclusive. The default value is 5.

### PR DESCRIPTION
**Pull request by JakubVanek**
Fixes #30

### CODE REVIEW

**Comment by krocheck** [comment]
Should this be a setting the user can set in the instance config?

**Comment by JakubVanek** [comment]
Updated so that this is now exposed in the configuration UI. I have also changed the default to 10, as I don't think that many people will want to see more broadcasts.

File: index.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ instance.prototype.init = function(is_config) {
 		self.config.client_secret,
 		self.config.client_redirect_url,
 		scopes,
+		parseInt(self.config.fetch_max_count) || 10,
```
Comment (krocheck) [inline index.js:50]: `parseInt` isn't necessary for a number field type.

Comment (JakubVanek) [reply index.js]: Fixed

**Pull request merged**